lms/djangoapps/courseware/module_render.py

```diff
@@ -475,13 +476,15 @@ def publish(block, event_type, event):
                 only_if_higher=event.get('only_if_higher'),
             )
         else:
-            aside_context = {}
+            context = contexts.course_context_from_course_id(course_id)
+            context['user_id'] = block.runtime.user_id
+            context['asides'] = {}
             for aside in block.runtime.get_asides(block):
                 if hasattr(aside, 'get_event_context'):
                     aside_event_info = aside.get_event_context(event_type, event)
                     if aside_event_info is not None:
-                        aside_context[aside.scope_ids.block_type] = aside_event_info
-            with tracker.get_tracker().context('asides', {'asides': aside_context}):
+                        context['asides'][aside.scope_ids.block_type] = aside_event_info
+            with tracker.get_tracker().context(event_type, context):
```

Comment:
I'll make that change. Definitely better to be safe.

I've validated that the context does get populated locally, and looks as we expect given the code here in module_render:

"context":{  
      "course_user_tags":{  

      },
      "user_id":5,
      "org_id":"edX",
      "asides":{  

      },
      "course_id":"course-v1:edX+DemoX+Demo_Course",
      "path":"/courses/course-v1:edX+DemoX+Demo_Course/courseware/9fca584977d04885bc911ea76a9ef29e/07bc32474380492cb34f76e5f9d9a135/"
   },